.circleci/config.yml

```diff
@@ -3320,6 +3320,14 @@ workflows:
               parallelism: [2]
               cluster-id-prefix: ["nightly"]
               mark: ["nightly"]
+      - test-e2e-aws:
```

Comment:
What do you think of putting quarantined tests like these in a quarantine workflow? The [CI triage recommendation](https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1306132589/CI+triage+runbook#CI-Quarantine-Suite) obviously wasn't authored with nightly in mind, and adding them to `test-e2e-quarantined` seems wrong, but it sucks to have the nightly workflow failing due to quarantined tests -- even if a moment's investigation shows the failing jobs as quarantined.